### ISSUE TITLE
Enable XMPP and IRC gateways on Slack

### ISSUE THREAD

**Issue by CodeNonprofit** [FreeCodeCamp/freecodecamp#312]
Slack accessibility options need to be turned on manually. See @Zersiax for more.

**Comment by zersiax** [comment]
To clarify, the slack web interface, like Gitter's, could be much more accessible. It is better in some ways, worse in others. The open XMPP or IRC protocols can be talked to by interfaces that are more familiar since a lot of people will for example already be using those same tools to work with Facebook Chat etc.
Per this article: https://slack.zendesk.com/hc/en-us/articles/201727913-Connecting-to-Slack-over-IRC-and-XMPP , it appears for security reasons, these gateways are disabled by default. Please consider enabling them, I think the security concerns can be called minimal at best.

**Comment by QuincyLarson** [comment]
@zersiax I agree 100%. Sorry it took me so long to get to this card. Both IRC and XMPP are now enabled. Enjoy!
